package.json

```diff
@@ -78,6 +78,7 @@
     "node-sword-interface": "^0.219.0",
     "officegen": "^0.6.4",
     "pug": "^3.0.0",
+    "remove-markdown": "^0.3.0",
```

Comment:
Just commenting ... the last published release of this package is almost three years ago. If it would be a major dependency I'd be worried that the project may be dead ... but since this is such a small support library I'm ok with this.

Reply:
Yes, I looked into that. It's just one file with a few regexes to remove markdown. If we need we can easily extend it in the future.